src/convertToObject.js

```diff
@@ -3,16 +3,30 @@
 /**
  * Implement convertToObject function:
  *
- * Function takes a string with styles (see an example in [stylesString.js](./stylesString.js))
+ * Function takes a string with styles
+ * (see an example in [stylesString.js](./stylesString.js))
  * and returns an object where CSS properties are keys
- * and values are the values of related CSS properties (see an exampl in [test file](./convertToObject.test.js))
+ * and values are the values of related CSS properties
+ * (see an exampl in [test file](./convertToObject.test.js))
  *
  * @param {string} sourceString
  *
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const originString = sourceString.replace(/\s\s+/g, ' ');
+  const styles = originString.split(';');
+  const validStyles = styles.filter(style => style.includes(':'));
+
+  const stylesToObject = validStyles.reduce((prev, style) => {
```

Comment:
there is no need to create this variables
```suggestion
  return sourceString
    .replace(/\s\s+/g, ' ')
    .split(';')
    .filter(style => style.includes(':'));
    .reduce((prev, style) => {
```